app/services/subscriptions/billing_service.rb

```diff
@@ -5,7 +5,11 @@ class BillingService < BaseService
     def call(billing_at: Time.current)
       @today = billing_at
 
-      billable_subscriptions.group_by(&:customer_id).each do |_customer_id, customer_subscriptions|
+      billable_subscriptions_per_customer = billable_subscriptions.group_by(&:customer_id)
+      total = billable_subscriptions_per_customer.count
+      Rails.logger.info "[#{self.class.name}] Billing #{total} customers"
```

Comment:
Perhaps a suggestion to wait for the tracing improvements (https://github.com/getlago/lago-api/pull/2357) so you could extend the current span with this attribute? 🤔 (that should make graphing afterwards easier)

Reply:
I'd 1000x prefer a metric available to make a bar chart! 👍 
It's rediscuss this when I return from holidays (in 2 weeks!)